src/TpetraLinearSystem.C

```diff
@@ -1174,7 +1179,9 @@ void dump_graph(const std::string& name, int counter, int proc, LinSys::Graph& g
   for(size_t i=0; i<myGlobalIndices.size(); ++i) {
     GlobalOrdinal rowGID = myGlobalIndices[i];
     std::vector<GlobalOrdinal> vIndices(graph.getNumEntriesInGlobalRow(rowGID));
-    Teuchos::ArrayView<GlobalOrdinal> colIndices(vIndices);
+    Tpetra::CrsGraph<>::nonconst_global_inds_host_view_type colIndices("col_indices", vIndices.size());
+    for (size_t col_idx = 0; col_idx < vIndices.size(); ++col_idx) 
+      colIndices(col_idx) = vIndices[col_idx];
```

Comment:
Same here, no need to loop a copy contents of vIndices. colIndices will be filled by the following call to graph.getGlobalRowCopy.

Reply:
Is there any sensitivity over this name, "col_indices"? We may have multiple equation systems sitting around and I wonder if this name needs to be specialized to the linear system? In general, most systems are the same size (1x1) with exception of the 3x3 momentum system. My guess is that the regression tests would have picked something up.